product-backend/scripts/01-createDataBase.sh

```diff
@@ -1,4 +1,6 @@
 mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "01-job-roles.sql"
-mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "02-registration-system-role.sql"
-mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "03-registration-system-user.sql"
-mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "04-capability.sql"
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "02-capabilities.sql"
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "03-registration-system-role.sql"
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "04-registration-system-user.sql"
+mysql -h"$DB_HOST" -D"$DB_NAME" -u"$DB_USERNAME" -p"$DB_PASSWORD" < "05-capability.sql"
```

Comment:
I removed the file 04-capabilite because it contains an incorrect instruction to create a table.

File 02-capabilities I added because it is responsible for creating Capabilities table and adds capabilities field to JobRoles table.